packages/fxa-auth-server/test/scripts/bulk-mailer.js

```diff
@@ -13,17 +13,17 @@ const fs = require('fs');
 const mocks = require(`${ROOT_DIR}/test/mocks`);
 const path = require('path');
 const rimraf = require('rimraf');
+const crypto = require('crypto');
 
 const cwd = path.resolve(__dirname, ROOT_DIR);
-cp.execAsync = promisify(cp.exec);
+const execAsync = promisify(cp.exec);
```

Comment:
There's still some references to cp.execAsync. Is this intentional? Or should everything be use `execAsync` now instead of `cp.execAsync`?`